x-pack/plugins/stack_alerts/server/alert_types/es_query/alert_type.test.ts

```diff
@@ -135,6 +135,7 @@ describe('alertType', () => {
 
     const searchResult: ESSearchResponse<unknown, {}> = generateResults([]);
     alertServices.scopedClusterClient.asCurrentUser.search.mockResolvedValueOnce(
+      // @ts-expect-error not compatible agregations type
```

Comment:
I'm not sure if this is in the current issue scope, but is it possible to have `elasticsearchClientMock` in the common folder, just to avoid the having annotation in the each file:
```
// eslint-disable-next-line @kbn/eslint/no-restricted-paths
import { elasticsearchClientMock } from '../../../../../../src/core/server/elasticsearch/client/mocks';
```

Reply:
It's provided from `'../../../../../../src/core/server/mocks`